packages/hardhat-verify/test/unit/etherscan.ts

```diff
@@ -99,7 +99,7 @@ describe("Etherscan", () => {
       assert.equal(currentChainConfig.chainId, 5);
     });
 
-    it("should throw if the selected network is hardhat and it's not a added to custom chains", async () => {
+    it("should throw if the selected network is hardhat and it's not an added to custom chains", async () => {
```

Comment:
This fix is incorrect. It should be something like "should throw if the selected network is hardhat and it's not added to custom chains". Could you update it?